src/hotspot/cpu/riscv/macroAssembler_riscv.hpp

```diff
@@ -188,15 +188,15 @@ class MacroAssembler: public Assembler {
   void access_load_at(BasicType type, DecoratorSet decorators, Register dst,
                       Address src, Register tmp1, Register thread_tmp);
   void access_store_at(BasicType type, DecoratorSet decorators, Address dst,
-                       Register src, Register tmp1, Register thread_tmp);
-  void load_klass(Register dst, Register src);
-  void store_klass(Register dst, Register src);
-  void cmp_klass(Register oop, Register trial_klass, Register tmp, Label &L);
-
-  void encode_klass_not_null(Register r);
-  void decode_klass_not_null(Register r);
-  void encode_klass_not_null(Register dst, Register src, Register tmp = xheapbase);
-  void decode_klass_not_null(Register dst, Register src, Register tmp = xheapbase);
+                       Register val, Register tmp1, Register tmp2, Register tmp3);
```

Comment:
I think we should not touch 'access_store_at' here?

Reply:
Oh sorry, I confused the other branches here when rebase. By the way, the correct code was used when testing.